i18n/ru.json

```diff
@@ -97,25 +97,25 @@
     "Add comment":
         "Добавить комментарий",
     "Optional nickname…":
-        "Опциональный никнейм…",
+        "Опциональный никнейм...",
```

Comment:
We [do use](https://github.com/PrivateBin/PrivateBin/blob/master/tpl/bootstrap.php#L9) Unicode, so there is no need to care about outdated codepages or such stuff…
"correct", because it is the proper symbol for `...`.